### DISCUSSION TITLE
CNCF OSPO BoF KubeCon India - Activity Structure & Planning

### DISCUSSION THREAD

**Discussion by anajsana** [todogroup/ospology#524] [Announcements]
# 🚀 Announcing the CNCF OSPO BoF at KubeCon + CloudNativeCon India | Dec 11th  

📢 We’re excited to announce the activity structure that will be used for the upcoming OSPO Birds of a Feather (BoF) session at [KubeCon + CloudNativeCon India](https://events.linuxfoundation.org/kubecon-cloudnativecon-india/program/experiences/#open-source-program-office-bof) on December 11th!  

## 🔗 Prepare in Advance  
Participants can review the official activity structure [here](https://github.com/user-attachments/files/17980518/OSPO_BoF_Activity.pdf)

> [!note]
> [OSPOlogy BoF Template, Adopters and Official Documentation](https://github.com/todogroup/ospology/tree/main/BoF)

## 📋 What to Expect  

- Identify the role of open source management in driving security, innovation, and collaboration within cloud-native ecosystems
- Identify solutions and best practices for leveraging open source management to optimize cloud-native infrastructure
- Facilitate knowledge sharing among participants to address challenges and uncover opportunities for growth

Using the OSPOlogy BoF framework, participants will collectively:  

- Frame Challenges
- Create an Affinity Mapping to identify common themes and group solutions.
- Summarize insights and foster actionable outputs to work async 

## 🕒 Session Details  

- Duration: 1 Hour  
- Date: December 11th  

> Space is limited, and we encourage early registration to ensure your participation. Click [here to RSVP](https://forms.gle/Qso8p2oMqK1kKn6n9) and secure your spot!

We look forward to seeing you there! 🎉

**Comment by ramanuj-dad** [comment]
It was an incredible event, and I thoroughly enjoyed it. The discussion topics, such as the challenges of running an Open Source Program Office (OSPO) and fostering open source contributions within commercial organizations, especially in the Indian context, were highly engaging and insightful.

I’m curious to know when the summary of discussions across all groups and the resulting action items for the community will be shared. Looking forward to collaborating further and building on these ideas!

**Reply by anajsana** [reply]
Thank you, @rockybhanu! I’ve just sent an email to all OSPO BoF registrants with further details on the next steps. I’ve also shared this link as an open forum where the community can share ideas, post updates on the progress, and will share the summary of the discussion once participants review the initial draft created.

**Comment by ansharora28** [comment]
The OSPOlogy BOF was a very interesting discussion. Indian organisations are very new or mostly unaware to the concept of OSPOs ( while there certainly are [exceptions](https://zerodha.com/z-connect/updates/the-ospo-leadership-award-to-zerodha) which others can learn from).

The draft doc looks good, and it pretty much covers everything I can recall from the clusters we were able to identify at the event.
It might be of interest to curate a list of Indian organisations that have active OSPOs and maybe reach out to them for understanding their background and challenges (and we can tell them about TODO group! )

Looking forward to contributing to the report.

**Reply by anajsana** [reply]
Thanks, @ansharora28! This idea has been incorporated into the [report draft](https://docs.google.com/document/d/11vxZqpuLEafZJz5LrphW2KoCuJ15_U6JniHpzFCmYDw/edit?usp=sharing) use as the collaborative tool to keep working async to finalize the report sections as an optional activity (see methodology section).

The ability to conduct the interviews may depend on the availability and interest of the contributors involved in creating this report to compile the list and identify someone to lead the effort of conducting the interviews as a volunteer, or explore potential sponsors

**Comment by anajsana** [comment]
## CNCF KubeCon India OSPO BoF Report Draft

The document attached serves as the collaborative tool used for contributing to the async work required to complete the [report](https://docs.google.com/document/d/11vxZqpuLEafZJz5LrphW2KoCuJ15_U6JniHpzFCmYDw/edit?usp=sharing).

People can check [the planning doc](https://docs.google.com/document/d/1n8H3d630PruhmSHbQliYRCEwyIiYYxsTc5Xf6426V7w/edit?usp=sharing) to learn more about the scope, expected outcomes, and timelines

### Background - BoF Activity Summary
https://todogroup.org/blog/bof-kubecon-india/ 

### Starting Point - Shared Challenges Discovered by OSPO BoF Participants 
The BoF session identified and prioritized the following topic clusters, focusing on challenges specific to the Indian market

| Topic                       | Challenges                                                                                      | “How might we” Questions                                                                                     |
|-----------------------------|------------------------------------------------------------------------------------------------|-------------------------------------------------------------------------------------------------------------|
| **Security**                | Ensuring robust security in Open Source projects.                                             | How might we address critical update event vulnerabilities in Open Source projects?                         |
| **Collaboration and Engagements** | Overcoming barriers to collaboration and improving networking among contributors.               | - How might we overcome networking issues that hinder productivity and collaboration?                       |
|                             |                                                                                               | - How might we enhance the process for sharing ideas effectively?                                           |
| **Starting and Structuring OSPOs** | Establishing and optimizing Open Source Program Offices (OSPOs).                              | - How might we structure the OSPO in the most efficient way?                                                |
|                             |                                                                                               | - How might we launch an OSPO with the right guidance and support?                                          |
| **Managing OSS Best Practices**    | Standardizing practices and addressing operational challenges in Open Source Software.        | - How might we standardize Open Source best practices in our company, including how employees engage with OSS projects and communities? |
|                             |                                                                                               | - How might we increase traction toward incubating projects like Kubernetes?                                |
|                             |                                                                                               | - How might we address security concerns, product bugs, and support requirements in OSS projects while balancing cost savings and losses? |
|                             |                                                                                               | - How might we encourage companies to contribute more to Open Source?                                       |
| **Product Roadmap / Engineering** | Aligning Open Source strategies with product development and engineering goals.               | - How might we navigate the challenges of moving away from upstream development, and why do we choose to fork? |
|                             |                                                                                               | - How might we enable developers to contribute effectively to Open Source?                                  |
|                             |                                                                                               | - How might we align different expectations from Open Source projects to avoid major bottlenecks?           |
|                             |                                                                                               | - How might we use a product roadmap to build an Open Source ecosystem and demonstrate returns on investment? |
